tests/integration-tests/tests/update/test_update.py

```diff
@@ -490,6 +495,40 @@ def _check_role_attached_policy(region, cluster, policy_arn):
     assert policy_arn in policies
 
 
+def get_ebs_volume_ids(cluster, region):
```

Comment:
Super minor: might be nice to have more descriptive function name, like `get_cfn_ebs_volume_ids`

Reply:
Done!